package.json

```diff
@@ -75,7 +75,8 @@
     "semantic-release": "^17.4.4",
     "semantic-release-gitmoji": "^1.3.4",
     "vue-code-highlight": "^0.7.8",
-    "vue-template-compiler": "2.6.14"
+    "vue-template-compiler": "2.6.14",
+    "jsdom": "^16.6.0"
```

Comment:
Jest uses JSDOM by default: https://jestjs.io/docs/26.x/configuration#testenvironment-string (unless we go Jest 27 which has new defaults but we are on 24 here).

It means that you don't need to install and import JSDOM by yourself. You can check this example from their docs: https://jestjs.io/docs/26.x/tutorial-jquery
It shows how it works with jQuery but it follows the same principles we need.

You can use `window` directly in the code as Jest will take care of setting it up for us (with JSDOM).

Let me know if you need help here!

Reply:
Done!